docs/quickstart-restore-backup.md

```diff
@@ -0,0 +1,258 @@
+# Restore the backup
+
+The following steps describe how to restore your backup to another Percona Server container and check whether the data is available.
+{.power-number}
+
+1. Create another Docker volume
+
+    ```{.bash data-prompt="$"}
+    $ docker volume create myvol2
+    ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        myvol2
+        ```
+
+2. Run another Percona Server container to restore the backup to
+
+    | Option | Description |
+    |---|---|
+    | `--name` | Provides a name to the container. If you do not use this option, Docker adds a random name. |
+    | `-d` | Detaches the container. The container runs in the background. |
+    | `-e` | Adds an environmental variable. This example adds the  ` MYSQL_ROOT_PASSWORD `  environment variable. The instance refuses to initialize if no environmental variable is added. Choose a more secure password, if needed. |
+    | `myvol2` | Indicates the persistent storage for the database. |
+    | `8.0.34` | Use this tag to specify a specific version. |
+
+    You must provide at least one environment variable to access the database, such as `MYSQL_ROOT_PASSWORD`, `MYSQL_DATABASE`, `MYSQL_USER`, and `MYSQL_PASSWORD` or the instance refuses to initialize.
+
+    For this document, we are using the `8.0.34` tag. In Docker, a tag is a label assigned to an image. Tags are used to maintain different versions of an image. If we did not add a tag, Docker uses `latest` as the default tag and downloads the latest image from [percona/percona-server on the Docker Hub].
+
+    To use a Docker volume for persistent storage with a database, specify the path where the database stores its data inside the container, usually `/var/lib/mysql`.
+
+    * Run a Docker container example
+
+        ```{.bash data-prompt="$"}
+        $ docker run -d -p 3307:3306 --name psmysql2 -e MYSQL_ROOT_PASSWORD=secret -v myvol2:/var/lib/mysql percona/percona-server:8.0.34
+        ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+
+        684464ce43d8880500f81abccb1e7440146a43f7d79c983a2a2a622b9a0d2080
+        ```
+ 
+3. Stop `psmysql2` container
+
+    ```{.bash data-prompt="$"}
+    $ docker stop psmysql2
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        psmysql2
+        ```
+
+4. Remove all created files in `myvol2` to allow xtrabackup restore data to `myvol2`
+
+    ```{.bash data-prompt="$"}
+    $ docker run --volumes-from psmysql2 -v backupvol:/backup_8034 -it --rm --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "rm -rf /var/lib/mysql/*"
+    ```
+
+    If the command executes successfully, the expected output is empty.
+
+    The `--rm` option automatically removes the temporary container created from percona/percona-xtrabackup:8.0.34 image after the container exits. 
```

Comment:
Move this information before the command is run.

Reply:
done